corehq/apps/reports/views.py

```diff
@@ -1175,7 +1192,10 @@ def _render_report_configs(request, configs, domain, owner_id, couch_user, email
         "email": email,
         "notes": notes,
         "unsub_link": get_url_base() + reverse('notification_unsubscribe', kwargs={
-            'scheduled_report_id': report_id, 'domain': domain}) if report_id else None,
+            'scheduled_report_id': report_id,
+            'user_email': couch_user.get_email(),
```

Comment:
This isn't the correct email to use. This is the email of the report owner, not necessarily the person receiving the report. Well either have to change it to render the report for each individual email or else render it with a place holder and then do a second format with just the unsub link.

See https://github.com/dimagi/commcare-hq/blob/d19f0237dbc6267dfda4230767a1f2b2f89a119d/corehq/apps/reports/models.py#L770-L774